File: frontend/src/api/useSystemApi/idCenter.ts
import request from '/@/utils/request';

/**
 * 菜单接口
 * @method getId 获取id
 */
export function useIdCenterApi() {
  return {
    // 获取id
    getId: () => {
      return request({
        url: '/idCenter/getId',
        method: 'GET',
        data: {},
      });
    },
  }
}

File: backend/autotest/apis/system/id_center.py
# -*- coding: utf-8 -*-
# @author: xiaobai
from fastapi import APIRouter

from autotest.utils.response.http_response import partner_success
from autotest.utils.snowflake import IDCenter

router = APIRouter()


@router.get('/getId')
def get_id():
    """
    获取id
    :return:
    """
    data = IDCenter.get_id()
    return partner_success(str(data))
